lib/chef-dk/policyfile_cache.rb

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright:: Copyright (c) 2014-2018 Chef Software Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require "chef-dk/exceptions"
+
+module ChefDK
+  module Policyfile
+    # Subclass of Chef's Cookbook::Metadata class that provides the API expected
+    # by CookbookOmnifetch
+    class Cache
```

Comment:
I don't think I need this class actually... this is leftover from a early iteration which didn't work out.